package-manifest.toml

```diff
@@ -569,10 +569,10 @@ only_for_targets.image = "standard"
 # 3. Use source.type = "manual" instead of "prebuilt"
 source.type = "prebuilt"
 source.repo = "crucible"
-source.commit = "e58ca3693cb9ce0438947beba10e97ee38a0966b"
+source.commit = "ea2282e24ecb50bf4aa2cbe069df10b80d96e162"
```

Comment:
Ideally the commit ID here should match what propolis has for crucible, so the two are in sync.
It's not always necessary, depending on what the change is though.

Reply:
Yeah, let me update this to match-up across the PRs.

Reply:
@leftwo I'll tie this to a crucible branch with temp deps (referencing this branch), and have it the same as propolis. Then, I'll go back and update accordingly once that merges.

Reply:
Ok. Updated.